compiler/src/dmd/cparse.d

```diff
@@ -2728,8 +2728,8 @@ final class CParser(AST) : Parser!AST
             case TKW.xident:
             {
                 const idx = previd.toString();
-                if (idx.length > 2 && idx[0] == '_' && idx[1] == '_')  // leading double underscore
-                    importBuiltins = true;  // probably one of those compiler extensions
+                if (idx.length > 1 && idx[0] == '_')  // leading underscore
+                    importBuiltins = true;  // maybe one of those compiler extensions
```

Comment:
I'm a little concerned about nearly always importing 10,000 lines of code for Windows compilations.

Reply:
Mhm, it's not great. How about what I've pushed now?
We have a lazily-initialised `StringTable` of all the MSVC intrinsic names that we have implemented.
If we're targeting the Microsoft C runtime, and we come across an identifier with a leading underscore, then we check if that identifier is a known MSVC intrinsic, and if it is we import the MSVC builtins and we stop checking future identifiers.

The downside there is we have to remember to update that `StringTable` if more MSVC intrinsics are implemented, and we do pay the minor cost of hashing and looking up identifiers with leading underscores.